src/chain/chain-generic-numerator.h

```diff
@@ -98,71 +96,79 @@ class GenericNumeratorComputation {
   bool Backward(CuMatrixBase<BaseFloat> *nnet_output_deriv);
 
  private:
-
   // Defining this constant as an enum is easier.  it controls a memory/speed
   // tradeoff, determining how many frames' worth of the transposed derivative
   // we store at a time.  It's not very critical; the only disadvantage from
   // setting it small is that we have to invoke an AddMat kernel more times.
   enum { kMaxDerivTimeSteps = 8 };
 
-  // sets up the alpha for frame t = 0.
-  void AlphaFirstFrame();
+  // for the remapped fsts, copy the appropriate activations to CPU memory
```

Comment:
I think this needs quite a bit more documentation.  And say what the 'remapped fsts' are, e.g. provide a pointer to where the concept is explained.